### PR TITLE
Remove qiskit.org links

### PR DESCRIPTION
**Pull request by Eric-Arellano**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -189,12 +189,6 @@ notes will look like for the current state of the repo you can run:
 and the release notes in particular will be located at
 `docs/_build/html/release_notes.html`
 
-## Installing Qiskit IBM Experiment from source
```
Comment (Eric-Arellano) [inline CONTRIBUTING.md:192]: These docs are long deleted. I'm not sure if you want something else to replace this?

**Comment by Eric-Arellano** [comment]
Gentle bump @gadial

File: CONTRIBUTING.md (hunk 1 above)
Comment (wshanks) [reply CONTRIBUTING.md:192]: Hmm, these documents probably need a broader cleanup. Maybe they should recommend tox like qiskit-experiments does.

**Review by wshanks — APPROVED** [review]
Looks good!

**Pull request merged**